curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md

```diff
@@ -30,6 +34,10 @@ tests:
     testString: assert(typeof fiboEvenSum(10) === 'number');
   - text: <code>fiboEvenSum(10)</code> should return 10.
     testString: assert.strictEqual(fiboEvenSum(10), 10);
+  - text: Your function should sum the even-valued Fibonacci numbers :<code>fiboEvenSum(8)</code> should return 10.
```

Comment:
```suggestion
  - text: 'Your function should sum the even-valued Fibonacci numbers: <code>fiboEvenSum(8)</code> should return 10.'
```
Sorry, @mainak-debnath, I had not noticed the `:` which will confuse the parser.  If you wrap the test in quotes (like this) it stops the parser from trying to interpret the `:`.

Reply:
@ojeytonwilliams  The above commit seems to be fine or should I change it

Reply:
@mainak-debnath, could you use this one?  It's nitpicky, but this has the space in the right place.  You can click 'Commit suggestion', rather than going though the command line, if you like.

Reply:
@ojeytonwilliams  Ok. No problem